Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="AutoConfiguredNSubstituteCustomization.cs" />
     <Compile Include="AutoNSubstituteCustomization.cs" />
+    <Compile Include="AutoSubstituteCustomization.cs" />

```

Comment:
> `AutoNSubstituteCustomization` has a constructor that takes an `ISpecimenBuilder` instance which is stored and returned by the `Builder` property. What is their intended use in consumer code?

The `Builder` property is an [Inspection Property](http://blog.ploeh.dk/2013/04/04/structural-inspection). It has no explicit intended use in consumer code, but (like all other Inspection Properties) exist as a courtesy to any caller that may want to inspect it.

When it comes to the constructor overload taking an `ISpecimenBuilder`, I can't remember the reason. Perhaps the Git log can answer that question...

> With the exception of the `SubstituteAttributeRelay`, none of the new classes call NSubstitute directly. Would you leave their names without the N?

That would make sense according to the above heuristic, but we don't need to make it a hard rule. The only thing that concerns me here is if we have two classes with *almost* identical names, because users are bound to mistype those from time to time.